CommandLine/Modular/Gradle/hellofx/build.gradle

```diff
@@ -1,30 +1,14 @@
-buildscript {
-    repositories {
-        mavenCentral()
-        maven {
-            url "https://plugins.gradle.org/m2/"
-        }
-    }
-    dependencies {
-        classpath 'com.google.gradle:osdetector-gradle-plugin:1.6.0'
-        classpath 'org.javamodularity:moduleplugin:1.1.1'
-    }
+plugins {
+  id 'application'
```

Comment:
Do we still need the application plugin?

Reply:
Yes, it was previously part of the JavaFX plugin, but it has been removed, so you need to include it.